cds-snc/c19-benefits-node
# Feature/remove feedback text

# Description

Add a feature flag FF_ENABLE_FEEDBACK_TEXT that will allow us to enable and disable the free text in the feedback 

Add a healthcheck page that will allow us to see what Feature flags are set.

![image](https://user-images.githubusercontent.com/87738/84532859-fd2e9280-acb4-11ea-8ba9-21a4747ef914.png)


## Test Instructions

1. Run branch locally
1. Set FF_ENABLE_FEEDBACK_TEXT to false
1. Verify that the free text is not enabled on feedback form
1. Attempt to send feedback see that 'disabled' gets added to the details list
1. SET FF_ENABLE_FEEDBACK_TEXT to true
1. Verify that free text is enabled on feedback form
